uber/common.py

```diff
@@ -57,8 +57,8 @@
 from sqlalchemy.sql.expression import FunctionElement
 from sqlalchemy.orm.attributes import get_history, instance_state
 from sqlalchemy.schema import Column, ForeignKey, UniqueConstraint
-from sqlalchemy.orm import Query, relationship, joinedload, backref
 from sqlalchemy.types import Boolean, Integer, Float, TypeDecorator, Date
+from sqlalchemy.orm import Query, relationship, joinedload, subqueryload, backref
```

Comment:
``subqueryload`` is something we're already doing in some of our other plugins, but apparently this is the first time that it's making its way here :/

That actually bodes well for optimizing all of the other slow pages, since it means there's some serious low hanging fruit to make things a lot faster.